Makefile

```diff
@@ -0,0 +1,16 @@
+CFLAGS:= -Wall -Wstrict-aliasing=1 -Wsign-conversion -I. -Isw_i2c
+
+OBJECTS = \
+	sensirion_common.o \
+	sw_i2c/sensirion_sw_i2c.o \
+	sw_i2c/sensirion_sw_i2c_implementation.o \
+	sw_i2c/sample-implementations/linux_user_space/sensirion_sw_i2c_implementation.o \
+	hw_i2c/sensirion_hw_i2c_implementation.o \
+	hw_i2c/sample-implementations/linux_user_space/sensirion_hw_i2c_implementation.o
```

Comment:
You only compile the linux user space sample implementation. Would it make sense to also compile the other sample implementations to check if they are correct?

Reply:
Because CI runs on Linux and not on the other platforms. We can't compile the other examples without cross-compile support

Reply:
Good point...